web/flat-web/deploy/nginx/nginx.conf

```diff
@@ -15,6 +15,19 @@ http {
     listen 80;
     root /app;
 
+    gzip on;
+    gzip_disable "MSIE [1-6]\.";
+    gzip_min_length 64;
+    gzip_buffers 16 8k;
+    gzip_comp_level 6;
+    gzip_static on;
```

Comment:
We need enable `--with-http_gzip_static_modul`. See: http://nginx.org/en/docs/http/ngx_http_gzip_static_module.html#gzip_static

Reply:
It looks like we don't need to make any additional changes, the `Nginx` image carries it by default.
<img width="1464" alt="image" src="https://user-images.githubusercontent.com/8198408/184607557-ff11600b-90ee-49f5-9f7c-0bfd483bfd5d.png">

Ref: https://github.com/nginxinc/docker-nginx/issues/14#issuecomment-71465684

Reply:
It seems that the default startup configuration of nginx on the server is already very nice